user-service/src/main/java/uk/gov/gchq/palisade/service/user/model/UserRequest.java

```diff
@@ -31,11 +31,10 @@
 import java.util.StringJoiner;
 
 /**
- * The UserRequest is the input for user-service to identify the User associated with the given User ID.
- * UserResponse is the output for this service which will include the User identified by the service.
+ * The UserRequest is the input for User Service, it contains a UserId value to identify the User associated with the request.
  * Note there are two classes that effectively represent the same data but represent a different stage of the process.
- * uk.gov.gchq.palisade.service.palisade.request.OriginalRequest is the client request that has come into the Palisade Service.
- * uk.gov.gchq.palisade.service.user.request.UserRequest is the input for the user-service.
+ * uk.gov.gchq.palisade.service.palisade.model.PalisadeSystemResponse is the system output from the Palisade Service, if successful.
+ * uk.gov.gchq.palisade.service.user.request.UserRequest is the input for the User Service.
```

Comment:
again, the input for the service is not very descriptive
```suggestion
 * uk.gov.gchq.palisade.service.user.request.UserRequest contains the request information from the Palisade Service which is required to find a valid User.
```

Reply:
Updated